examples/pytorch/semantic-segmentation/run_semantic_segmentation.py

```diff
@@ -108,7 +108,7 @@ class DataTrainingArguments:
             )
         },
     )
-    reduce_labels: Optional[bool] = field(
+    do_reduce_labels: Optional[bool] = field(
```

Comment:
For the script, we'll need to deprecate `reduce_labels` e.g. [like here](https://github.com/huggingface/transformers/blob/aada568f73953d0678bbb0d605bfeed46fb209fd/examples/flax/token-classification/run_flax_ner.py#L399)